src/v2/view-builder/views/IdentifierView.js

```diff
@@ -18,6 +18,14 @@ const Body = BaseForm.extend({
     return loc('oform.next', 'login');
   },
 
+  initialize() {
+    BaseForm.prototype.initialize.apply(this, arguments);
+
+    if (this.getUISchema().find(schema => schema.name === 'credentials.passcode')) {
+      this.save = loc('oie.identifier.form.sign.in', 'login');
```

Comment:
how about passing argument to `save(name)` and do this login in `save` ? Here just call `this.save(name)`

Reply:
Calling save does not update the save button text.